cmd/res/configuration.toml

```diff
@@ -14,7 +21,7 @@ ConnectRetries = 10
 Labels = []
 EnableAsyncReadings = true
 AsyncBufferSize = 16
-UseMessageBus = false
+UseMessageBus = true
```

Comment:
Is this for demo purpose since other device services are defaulting to `false` ?

Reply:
Fixed